src/openms/include/OpenMS/TRANSFORMATIONS/FEATUREFINDER/MultiplexDeltaMassesGenerator.h

```diff
@@ -63,6 +65,7 @@ namespace OpenMS
      * 
      * @param labels    isotopic labels
      * @param missed_cleavages    maximum number of missed cleavages due to incomplete digestion
+     * @param label_mass_shift    name of labels and their corresponding mass shifts
      * For example due to knock-outs in one of the samples.
      */
     MultiplexDeltaMassesGenerator(String labels, int missed_cleavages, std::map<String,double> label_mass_shift);
```

Comment:
"For a detailed description and further examples see the algorithm::labels parameter in the FeatureFinderMultiplex und MultiplexResolver tools."
It needs to be documented here!